CHANGES.rst

```diff
@@ -14,6 +14,8 @@ Changelog for Isso
   to be added, see `Gravatar: Image requests`_.
 - Add ``data-isso-reply-notifications-default-enabled`` option to configure
   whether or not the subscribe to replies checkbox should be checked by default.
+- Add `data-isso-page-author-hashes` option to client which makes it possible
```

Comment:
`reST` requires double backticks (\`\`data-isso-...\`\`) around code instead of markdown's one.